peer/brontide.go

```diff
@@ -568,7 +574,7 @@ func NewBrontide(cfg Config) *Brontide {
 		return uint16(
 			// We don't need cryptographic randomness here.
 			/* #nosec */
-			rand.Intn(lnwire.MaxPongBytes + 1),
+			rand.Intn(PongSizeCeiling) + 1,
```

Comment:
Since we are here maybe we can add a full stop to the comment here: https://github.com/lightningnetwork/lnd/blob/52b677b1346b510c027faf97fc5c2419847bf8d5/peer/brontide.go#L572